test/lib/__snapshots__/api.spec.jsx.snap

```diff
@@ -25,6 +25,8 @@ exports[`api spec should set content of the store via declarative api 3`] = `
 </div>
 `;
 
+exports[`api spec should set content of the store via declarative api 4`] = `<div />`;
```

Comment:
what does that test ? ^^

Reply:
I had forgotten to put a content for the search. Fixed in https://github.com/cozy/cozy-bar/pull/706/commits/789de16c4f2c8064a9a79562945252a6a9a7571d